src/qt/font.cpp

```diff
@@ -469,9 +469,20 @@ void wxNativeFontInfo::SetPixelSize(const wxSize& size)
 
 void wxNativeFontInfo::SetStyle(wxFontStyle style)
 {
-    m_qtFont.setItalic(style == wxFONTSTYLE_ITALIC);
-//case wxFONTSTYLE_SLANT:
-//case wxFONTSTYLE_NORMAL:
+    switch(style)
+    {
+        case( wxFONTSTYLE_ITALIC ):
+            m_qtFont.setItalic(true);
```

Comment:
Is there any difference between doing this and calling `setStyle(QFont::StyleItalic)`? If not, maybe we should prefer the latter, for consistency?

Reply:
I agree. There's no difference between the two, <setItalic> will ultimately call on <setStyle> so yes, better to have consistency.